docs/user_guide_ova.md

```diff
@@ -0,0 +1,190 @@
+#User Guide of Harbor Virtual Appliance
+##Overview
+This guide walks you through the fundamentals of using Harbor virtual appliance. You'll learn how to use Harbor to:  
+
+* Manage your projects.
+* Manage members of a project.
+* Replicate projects to a remote registry.
+* Search projects and repositories.
+* Manage Harbor system if you are the system administrator:
+ + Manage users.
+ + Manage destinations.
+ + Manage replication policies.
+* Pull and push images using Docker client.
+* Delete repositories and images.
+
+
+##Role Based Access Control
+
+![rbac](img/rbac.png)
+
+Harbor manages images through projects. Users can be added into one project as a member with different privileges. RBAC (Role Based Access Control) is provided in Harbor and there are four roles with different privileges:  
+
+* **Guest**: Guest has read-only privilege for a specified project.
+* **Developer**: Developer has read and write privileges for a project.
+* **ProjectAdmin**: When creating a new project, you will be assigned the "ProjectAdmin" role to the project. Besides read-write privileges, the "ProjectAdmin" also has some management privileges, such as adding and removing members.
+* **SysAdmin**: "SysAdmin" has the most privileges. In addition to the privileges mentioned above, "SysAdmin" can also list all projects, set an ordinary user as administrator and delete users. The public project "library" is also owned by the administrator.  
+* **Anonymous**: When a user is not logged in, the user is considered as an "anonymous" user. An anonymous user has no access to private projects and has read-only access to public projects.  
+
+##User account
+Harbor supports two authentication mode:  
+
+* **Database**  
+
+	Users are stored in the local database.  
+	
+	You can determine whether the self-registration is allowed or not under this authentication mode by setting **"Self Registration"** according to the [installation guide](installation_guide_ova.md). If this flag is set to off users can only be added by the system administrators.  

```

Comment:
Change to:
A user can self register himself/herself in Harbor in this mode. To disable user self-registration, refer to the [installation guide](installation_guide_ova.md). When self-registration is disabled, the system administrator can add users in Harbor.